triage/README.md

```diff
@@ -10,13 +10,13 @@ usage.
 - rylev
 - pnkfelix
 
-Monday evening in North America is a good time to do it because This Week in Rust (see below) is usually
-published on Tuesday, US time, and so it means the PR to include the triage
+Tuesday afternoon in North America is a good time to do it because This Week in Rust (see below) is usually
```

Comment:
I think maybe we can say something like Monday evening to Tuesday afternoon are good times to do triage? I know I usually do it at ~10am ET, for example.

Reply:
Done!